components/reusable/SpinText.tsx

```diff
@@ -0,0 +1,13 @@
+import { CircularText } from './CircularText';
+
+const SpinText = () => {
+  return (
+    <div className=" flex items-center justify-center">
+      <div className="relative flex h-auto w-[120px] items-center justify-center">
```

Comment:
Maybe you can try combining your divs here? You're only returning one thing that I see.